course/page/code.py

```diff
@@ -378,6 +350,9 @@ def sanitize_from_code_html(s):
                                         "img"],
             attributes=filter_from_code_html_attributes,
 
+            # strip unwanted tags
+            strip=True,
+
```

Comment:
As opposed to escaping them? Why?

Reply:
https://github.com/mozilla/bleach/issues/348#issuecomment-367337250

Reply:
Do you think `&lt;style src="data:,Evilcss"&gt;&lt;/style&gt;` still malicious?